docs/docs/bootstrapping.md

```diff
@@ -14,20 +14,30 @@ option                                        | description
 --where WHERE_CLAUSE                          | where clause to restrict the rows bootstrapped from the specified table
 --client_id CLIENT_ID                         | specify which maxwell instance should perform the bootstrap operation
 
-### Using the maxwell.bootstrap table
+### Starting a table bootstrap
 ***
-Alternatively you can insert a row in the `maxwell.bootstrap` table to trigger a bootstrap.
+You can start a bootstrap using:
 
 ```
-mysql> insert into maxwell.bootstrap (database_name, table_name) values ('fooDB', 'barTable');
+bin/maxwell-bootstrap --config localhost.properties --database fooDB --table barTable --log_level info
 ```
+
 Optionally, you can include a where clause to replay part of the data.
 
-bin/maxwell-bootstrap --config localhost.properties --database foobar --table test --log_level info
+```
+bin/maxwell-bootstrap --config localhost.properties --database fooDB --table barTable --where "my_date >= '2017-01-07 00:00:00'" --log_level info
```

Comment:
here too, `--log_level` and `--config` are just clutter

Reply:
Done